README.md

```diff
@@ -1,3 +1,6 @@
+__NOTE:__ We have merged this plugin into [Observability Plugin](https://github.com/opensearch-project/trace-analytics/) since OpenSearch 1.2, please use notebooks in Observability Dashboards plugin. Thanks.
```

Comment:
np- the "Thanks" appended at the end seems unnecessary/almost passive-aggressive?

Reply:
oh, removing it...